src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -79,19 +93,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $this->drop        = $input->getOption('drop');
         $this->skip_check  = $input->getOption('skip-exists-check');
+        $this->existent_db = $input->getOption('use-existent-db');
 
         $this->check($input, $output);
 
         if ($this->drop) {
             $this->_executeSQL(sprintf("DROP DATABASE IF EXISTS `%s`", $this->target_db));
         }
 
-        $result = $this->_executeSQL(sprintf("CREATE DATABASE IF NOT EXISTS `%s` CHARACTER SET utf8", $this->target_db));
-
-        if (!empty($result)) {
-            throw new \RuntimeException(sprintf('Cannot create database %s. Error: %s', $this->target_db, $result));
+        if (!$this->existent_db) {
+            $result = $this->_executeSQL(sprintf("CREATE DATABASE IF NOT EXISTS `%s` CHARACTER SET utf8", $this->target_db));
```

Comment:
@levnikmyskin The query to create the database has an explicit `IF NOT EXISTS` statement in it, which means it shouldn't do anything if the database already exists. So we shouldn't need the if statement here. Did you run into trouble with this query in particular?

Reply:
Hi,
Yes, the reason of this PR was mainly related to this query. I work for an hosting reseller company and we don't give our customers root access on the database, but rather we create the database, create the user and set it so that this user can only access that database (this is pretty common for managed hostings, customers don't really want to mess things up with root access).
That's why if I want to use joomlatools in a script I cannot use root.
That said, I still want to install joomla in an already existent and empty database with a given user who has read/write access on that db. This option is allowing exactly this: if you already have a database and an user for that database you can use it to install joomla, no root required. 
If you need more info, please ask :)

Reply:
@levnikmyskin OK, I see. If you don't give the `CREATE` privilege to the user, this statement will fail indeed. I'd like to rename the option to `skip-create-statement` though, that makes it much more clear what it does. Would you mind updating that please?

Reply:
Sure, renamed it in one of the latest commits